### DISCUSSION TITLE
admin password reset

### DISCUSSION THREAD

**Discussion by wesley-pearson** [alfio-event/alf.io#1003] [Q&A]
Hello! 

Just deployed a Digital Ocean Droplet to showcase to my team. 

I was uncle to get the admin password from the initial set up. 

Is there a way to reset?

**Comment by cbellone** [comment]
Hi! 
You should have the password in the logs under /home/alfio/logs if you have created the droplet from the marketplace

Right now, admin password cannot be reset without accessing the application. 

The only workaround is:

- create a new postgres database
- start alf.io pointing to the new database (name: d2, you can modify the application.properties to achieve this)
- copy the generated password from the log
- copy the encrypted password from d2 to the original database (table is `ba_user`)
- start alf.io pointing to the original database
